dojo/models.py

```diff
@@ -1098,10 +1111,10 @@ def compute_hash_code(self):
             hash_string = hash_string.strip()
             return hashlib.sha256(hash_string).hexdigest()
 
-
     def duplicate_finding_set(self):
         return self.duplicate_list.all().order_by('title')
 
+
```

Comment:
Shall we remove this change if it is not an intended correction?

Reply:
Removed.